firebase/functions/src/post-user-create.ts

```diff
@@ -1,11 +1,12 @@
-import * as functions from 'firebase-functions';
-import { TriggerAnnotated, Event } from 'firebase-functions';
-import { DeltaSnapshot } from 'firebase-functions/lib/providers/database';
+import { database, config } from 'firebase-functions';
+
+import { Change, Runnable, TriggerAnnotated } from 'firebase-functions/lib/cloud-functions';
+import { DataSnapshot } from 'firebase-functions/lib/providers/database';
 
 import { setBetaPlan } from './user/add-plan';
 import { MailChimp } from './mailchimp/mailchimp';
 
-const mailchimpConfig = functions.config().mailchimp;
+const mailchimpConfig = config().mailchimp;
```

Comment:
Now I'm even more confused regarding the new config handling as this seems to indicate that the old way of handling configs is still here just that `functions.config()` moved to a single importable `config()`.

Reply:
Actually, that change is unrelated to it. We could as well go with import * as firebase and continue to use `firebase.config().mailchimp`. This is just a left over of me trying to debug something I had to fight with. So this can be ignored.